helpers/helpers.go

```diff
@@ -9,15 +9,22 @@ import (
 	"strings"
```

Comment:
Avoid meaningless package names like util, common, misc, api, types, and interfaces. See https://go.dev/doc/effective_go#package-names and https://go.dev/blog/package-names for more.

----

I assume that the package named `helpers` could also benefit from a more specific and meaningful name.

Reply:
it's not me who created package `helpers`. and it doesn't relate to the current PR. but I can rename it of course. please suggest a new name.

Reply:
let's keep it for now and not rename it.